src/patternfly/components/Divider/divider.hbs

```diff
@@ -2,6 +2,7 @@
 {{#if divider--attribute}}
   {{divider--attribute}}
 {{/if}}
+aria-hidden="true"
```

Comment:
did you mean to add this?

Reply:
I did, but maybe better addressed in another issue :)